src/rdf4cpp/storage/NodeStorage.cpp

```diff
@@ -0,0 +1,14 @@
+#include "NodeStorage.hpp"
+
+#include <rdf4cpp/storage/reference_node_storage/SyncReferenceNodeStorage.hpp>
+
+namespace rdf4cpp::storage {
+
+reference_node_storage::SyncReferenceNodeStorage default_node_storage_holder_;
```

Comment:
maybe wrap `#ifndef DOXYGEN_PARSER` around these 2 lines, breathe/exhale is struggling a bit with it